### DISCUSSION TITLE
how to move plugin resources in the vendor to filament resources

### DISCUSSION THREAD

**Discussion by ehary-devs** [filamentphp/filament#9881] [Help]
how to move plugin resources in the vendor to filament resources because I want to add something, for example changing the navigation group

**Comment by demobiel** [comment]
I would extend the resource and overwrite the properties I need.

**Reply by devhoussam1998** [reply]
first of all the plugin you use should explain the ability to custom certain stuff on it moreover you can ask about that in plugin official repository.

**Comment by leandrocfe** [comment]
What plugin? Could explain more about it please?
